easybuild/easyconfigs/t/tabix/tabix-0.2.6-foss-2019b.eb

```
@@ -0,0 +1,37 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'tabix'
+version = '0.2.6'
+
+homepage = 'http://samtools.sourceforge.net'
+description = """ Generic indexer for TAB-delimited genome position files """
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = [('https://sourceforge.net/projects/samtools/files/', 'download')]
+sources = [SOURCE_TAR_BZ2]
+checksums = ['e4066be7101bae83bec62bc2bc6917013f6c2875b66eb5055fbb013488d68b73']
+
```

Once this easyconfig is switched to `GCCcore` it will need this additional dependency
```suggestion

builddependencies = [('binutils', '2.32')]

```

---

Thanks for your help! I have switched to GCCcore-8.3.0 and added GCCcore-10.2.0 as some time has passed. 
Happy to only keep one of the two.